addons/l10n_cl/data/account_fiscal_template.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="afpt_non_recoverable_vat_1" model="account.fiscal.position.template">
+            <field name="name">Compras - destinadas a generar operaciones no gravadas o exentas</field>
```

Comment:
Names should be in english and translated in the `.po` file to the localization language. It is easier to us to maintain (we all speak english).